extracted/konan.serializer/src/org/jetbrains/kotlin/serialization/konan/interop/InteropUtils.kt

```diff
@@ -0,0 +1,59 @@
+package org.jetbrains.kotlin.serialization.konan.interop
+
+import org.jetbrains.kotlin.descriptors.ModuleDescriptor
+import org.jetbrains.kotlin.descriptors.PackageFragmentDescriptor
+import org.jetbrains.kotlin.descriptors.konan.interop.InteropFqNames
+import org.jetbrains.kotlin.konan.library.KonanLibraryReader
+import org.jetbrains.kotlin.name.FqName
+import org.jetbrains.kotlin.serialization.konan.KonanPackageFragment
+
+fun createInteropLibrary(reader: KonanLibraryReader): InteropLibrary? {
```

Comment:
Why this one goes to Kotlin? It has nothing to do with it.

Reply:
It's needed for reading interop KLIBs.

For example, `InteropLibraryImpl` provides `ExportedForwardDeclarationsPackageFragmentDescriptor` for every package fragment in KLIB.

Reply:
This shall be somehow rethought. This code does not belong to Kotlin at all.